apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-description/edit.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { PlainText } from '@wordpress/block-editor';
```

Comment:
We probably should use the older `editor` package until support is dropped. This will allow for Gutenberg support across more versions

Reply:
Oh, will do, good idea.

Reply:
https://github.com/Automattic/wp-calypso/pull/34096